src/test-apps/TestEventLogging.cpp

```diff
@@ -2034,6 +2033,8 @@ static void CheckGapDetection(nlTestSuite *inSuite, void *inContext)
     event_id_t eventId_A = nl::Weave::Profiles::DataManagement::LogEvent(testSchema, nl::SerializedDataToTLVWriterHelper, (void *)&appData);
     event_id_t eventId_B = nl::Weave::Profiles::DataManagement::LogEvent(testSchema, nl::SerializedDataToTLVWriterHelper, (void *)&appData);
 
+    (void)eventId_B;
```

Comment:
Should we use IgnoreUnusedVariable() for this?

Reply:
Happy to use that.